src/tidecli/utils/file_handler.py

```diff
@@ -349,8 +456,8 @@ def validate_answer_file(answer_by: list[str], metadata_by: list[str]) -> bool:
     than the task content lines. The task content lines are marked with
     comments/messages
 
-    :param answerfile: Path to the answer file
-    :param metadata_taskfile: Path to the metadata task file
+    :param answer_by: list of answers
+    :param metadata_by: list of metedatas
```

Comment:
typo, obscure by in name

Reply:
This is "orginila" code and if changes needed, it is better done by code "owner" when
he thinks the real usage.

Reply:
Right, "answer_by" already exists as a variable name. Thanks for the clarification. No need to address here then.